numba/npyufunc/dufunc.py

```diff
@@ -52,6 +63,10 @@ def __init__(self, py_func, **kws):
             kws.pop('identity', None))
         super(DUFunc, self).__init__(dispatcher, **kws)
         self._install_type()
+        self._kernel = type('DUFuncKernel_' + self.ufunc.__name__,
```

Comment:
I know it's just a paste from the previous PR, but for the record, why don't you declare the class in the usual way? It would make the code more readable...